test/cql-pytest/test_key_length.py

```diff
@@ -0,0 +1,257 @@
+# Copyright 2023-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+#############################################################################
+# Although regular column values are only limited by 32 bits (actually, 2GB),
```

Comment:
> 32 bits (actually, 2GB)

There's a huge difference between 32b and 2GB, could it be further explained in the comment why on one hand there's a 32bit limit and 2GB OTOH?

Reply:
Cassandra was written in Java and uses signed integers in most of the places. I didn't investigate too much why they didn't used an unsigned integer for a column length, I blindly believed the documentation (e.g., https://docs.datastax.com/en/cql-oss/3.x/cql/cql_reference/refLimits.html) that this is the limit. In this test file I'm interesting in the much lower 16-bit length limits.

Reply:
Changed the phrasing.